recipes/web.py/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "web.py" %}
+{% set version = "0.38" %}
+{% set sha256 = "09c61b5b6b6e1dd558818d07dfbab3b2da0623603079c7d7223644b287f08870" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --record record.txt
```

Comment:
Nevermind. I missed that you tried that before and I was looking at the source code on Github rather than the older version of `setup.py` that's in the tarball for 0.38